src/external/dftk_to_wannier90.jl

```diff
@@ -0,0 +1,605 @@
+using ProgressMeter
+using LinearAlgebra
+using Dates
+
+## Works in the following order
+# $ dftk2wan_win_file(...) to generate the win file
+# $ wannier90.x -pp "prefix" ------------ nnkp file
+# $ dftk2wan_wannierization_files(...)--- mmn,amn,eig files
+# $ wannier90.x "prefix"  --------------- actual wannierization
+
+## Remark : the routines "dict_AOs", "guess_win", "cart_to_spherical" and "A_k_matrix_win_guesses" and the "projs" table in "read_nnkp_file"
+# are dedicated to computation of A_k matrices with Wannier90 guess, read from the nnkp file.
+# It is bugged and may not be usefull but I left it there.
+
+## TODO
+# 1 - check if kgrid can be removed from dftk2wan_win_file arguments
+# 2 - add handling of the SCDM routine as an alternative guess
+# 3 - optimize the "generate_mmn" routine
+
+
+"""
+    Create a .win file for Wannier90, compatible with the system studied with DFTK.
+    Options have to be compeleted by hand.
+"""
+function dftk2wan_win_file(prefix::String,basis::PlaneWaveBasis,scfres,num_wann;
+                           kgrid = [4,4,4], #can we retrieve this from scfres or basis ? Feel this should not be an argument.
```

Comment:
The win file asks for the k grid in the specific format : 
mp_grid : 4 4 4 (or other grids)
That's the only reason I ask for the kgrid. But if it is possible to retrieve the "4 4 4" in some way, or if we decide not to generate a win file after all this will be obsolete !